db-service/lib/cql-functions.js

```diff
@@ -14,18 +14,6 @@ const StandardFunctions = {
    * @returns {string}
    */
   average: x => `avg(${x})`,
-  /**
-   * Generates SQL statement that produces a boolean value indicating whether the search term is contained in the given columns
-   * @param {string} ref
-   * @param {string} arg
-   * @returns {string}
-   */
-  search: function (ref, arg) {
-    if (!('val' in arg)) throw `SQLite only supports single value arguments for $search`
```

Comment:
Why don't you make the error here more generic? It shares the implementation for all databases.

Reply:
I thought of supporting search with multiple args for postgres as shown in #132 so separating the functions seemed like a good first step

Reply:
... applied your suggestion :)